rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -646,6 +646,14 @@ if [ -f ${TMP_CRON} ]; then
 fi
 %endif
 
+# Try to send an inventory after upgrade to see the new agent version on the server
+if [ ${CFRUDDER_FIRST_INSTALL} -ne 1 ]
+then
+  echo "INFO: Trigger an inventory..."
+  /var/rudder/cfengine-community/bin/cf-agent -KD force_inventory >> %{rudderlogdir}/install/rudder-agent.log 2>&1
```

Comment:
I assume this will be changed to "rudder agent inventory" in versions where that command exists?